repository/src/main/resources/db/changelog/db.changelog-rdm-2917.xml

```diff
@@ -0,0 +1,504 @@
+        <createTable tableName="case_type_acl">
+            <column name="id" type="serial">
+                <constraints nullable="false"/>
+            </column>
+            <column name="case_type_id" type="integer">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user_role_id" type="integer">
+                <constraints nullable="true"/>
+            </column>
+            <column name="create" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="read" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="update" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="delete" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="live_from" type="date" >
+                <constraints nullable="true"/>
+            </column>
+            <column name="live_to" type="date" >
+                <constraints nullable="true"/>
+            </column>
+            <column name="created_at" type="datetime" defaultValue="now()">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <addPrimaryKey columnNames="id"
+                       constraintName="pk_case_type_acl"
+                       tableName="case_type_acl"/>
+
+        <!-- Add foreign key for case_type_id column linking to case_type table -->
+        <addForeignKeyConstraint baseColumnNames="case_type_id"
+                                 baseTableName="case_type_acl"
+                                 constraintName="fk_case_type_acl_case_type_id"
+                                 referencedColumnNames="id"
+                                 referencedTableName="case_type"/>
+
+        <!-- Add foreign key for user_role_id column linking to user_role table -->
+        <addForeignKeyConstraint baseColumnNames="user_role_id"
+                                 baseTableName="case_type_acl"
+                                 constraintName="fk_case_type_acl_user_role_id"
+                                 referencedColumnNames="id"
+                                 referencedTableName="user_role"/>
+
+        <!-- Add unique constraint on case_type_id and user_role_id columns for the case_type_acl table -->
+        <addUniqueConstraint columnNames="case_type_id, user_role_id"
+                             constraintName="unique_case_type_acl_case_type_id_user_role_id"
+                             tableName="case_type_acl"/>
+
+        <!-- copy table data to the new case_type_acl table -->
+        <sql>
+            insert into case_type_acl select * from case_type_user_role;
+        </sql>
+
+        <!-- Add new role_id column to the case_type_acl table -->
+        <addColumn tableName="case_type_acl">
```

Comment:
I don't see the benefit of splitting this column creation from the table creation.